src/rpc/rpcevo.cpp

```diff
@@ -792,15 +811,29 @@ UniValue bls_generate(const JSONRPCRequest& request)
     return ret;
 }
 
-UniValue _bls(const JSONRPCRequest& request)
+void _bls_help()
```

Comment:
Let's maybe call this `bls_help()` similar to `gobject_help()`: https://github.com/dashpay/dash/blob/70eb710b12e1cbcc28677e9ab1ef90bfc7a0b611/src/rpc/governance.cpp#L948

In some languages, starting a method with `_` is indented to indicate private methods (of course c++ has true access specifiers, but that's beside the point), and generally not a great method name IMO. This will also be more uniform w/the other "subcommand" commands.